### PR TITLE
feat(labware-creator): change preview text to reflect labware type

### LINKED ISSUE #8028: bug(LC): copy error
1.  In labware creator -> check your work section.
2. SVG displays: Check that the size, spacing, and shape of your wells looks correct. instead of "Check that the size, spacing, and shape of your tips looks correct."
3. tips instead of wells

[Edit, after talking with Morgan and Neha the text should be as follows]
When labware type is tip rack, it should say "tips"
When labware type is tube rack, it should say "tubes"
Otherwise it should say "wells"

### PR DESCRIPTION
**Pull request by shlokamin**
# Overview

closes #8028

# Review requests
Code review

Create a tiprack, tuberack, and anything with wells. Check the preview section at the end to make sure the text dynamically reflects the labware type as per the ticket. 

# Risk assessment
Low

### CODE REVIEW

File: labware-library/src/labware-creator/components/sections/Preview.tsx [hunk 1]
```diff
@@ -1,13 +1,36 @@
...
+    case 'tubeRack':
+      labwarePreviewType = 'tubes'
+      break
+    default:
+      labwarePreviewType = 'wells'
+  }
+  return (
+    <p className={styles.preview_instructions}>
+      Check that the size, spacing, and shape of your {labwarePreviewType} looks
+      correct.
```
Comment (smb2268) [inline labware-library/src/labware-creator/components/sections/Preview.tsx]: I made a util `getLabwareName` that you should be able to use here to simplify this a bit: 
```
import { getLabwareName } from '../../utils'
const { values } = props
const labwarePreviewType = getLabwareName(values, true)
```

Comment (shlokamin) [reply labware-library/src/labware-creator/components/sections/Preview.tsx]: oh great, will swap this out. thx!

**Review by IanLondon — APPROVED** [review]
📖

**Review by smb2268 — APPROVED** [review]
🎉

**Pull request merged**